app/main.py

```diff
@@ -2,0 +1,18 @@
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list[dict], cafe: Cafe) -> str:
+    masks_to_buy = 0
+    for visitor in friends:
+        try:
+            cafe.visit_cafe(visitor=visitor)
+        except VaccineError:
+            return "All friends should be vaccinated"
+        except NotWearingMaskError:
+            masks_to_buy += 1
+
+    if masks_to_buy:
+        return f"Friends should buy {masks_to_buy} masks"
```

Comment:
Code Efficiency: There's no need to compare `masks_to_buy` to `0` in the `if` condition. In Python, `0`, empty containers (like `[]`, `{}`, `''`, etc.), `None`, and `False` are considered `False` in a boolean context. All other values are considered `True`. So you can simplify the `if` condition to:

```python
if masks_to_buy:
```
This is more Pythonic and easier to read.

Reply:
I already use this structure